packages/node/src/authenticatedFetch/fetchFactory.ts

```diff
@@ -81,8 +82,10 @@ export function buildBearerFetch(
       currentAccessToken = tokenSet.access_token;
       if (tokenSet.refresh_token) {
         // If the refresh token is rotated, update it in the closure.
-        // TODO: Plug this into external storage, if one is provided.
         currentRefreshOptions.refreshToken = tokenSet.refresh_token;
+        if (currentRefreshOptions.handleRefreshToken !== undefined) {
```

Comment:
A positive check on the type is safer (also down below):

```suggestion
        if (typeof currentRefreshOptions.handleRefreshToken === "function") {
```

Reply:
99010293006619e1288c0791b81a55a5c7ace116